NOTICE.txt

```diff
@@ -32494,6 +32471,29 @@ NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
 SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 
+--------------------------------------------------------------------------------
+Module  : gotest.tools
```

Comment:
Is the license still around because it's a transitive dependency?

Reply:
That's from: https://github.com/golang/go/issues/33008
because it is not getting pruned from go.sum properly. We can "fix" it by deleting go.sum and rerunning `go mod tidy` to recreate it. I was hesitant to do that but I think it might make sense

Reply:
Hmm I actually may have been wrong, I wonder if it's a dependency of a dependency? If I `rm go.sum && go mod tidy`, the regenerated go.sum still has:

>gotest.tools v2.2.0+incompatible h1:VsBPFP1AI068pPrMxtb/S8Zkgf9xEmTLJjfM+P5UIEo=
gotest.tools v2.2.0+incompatible/go.mod h1:DsYFclhRJ6vuDpmuTbkuFWG+y2sxOXAzmJt81HFBacw=

But I think either I'm confused what go.sum is for or `go mod tidy` is

>12:08 $ go mod why gotest.tools
# gotest.tools
(main module does not need package gotest.tools)

`go.mod` should have all direct and indirect dependencies going by the docs: https://github.com/golang/go/wiki/Modules#why-does-go-mod-tidy-record-indirect-and-test-dependencies-in-my-gomod

so I'm not sure why it's adding that to go.sum